src/Backend/OnboardingInsuranceAPI/Areas/User/UserData.cs

```diff
@@ -0,0 +1,27 @@
+using System.Text.Json.Serialization;
+
+namespace OnboardingInsuranceAPI.Areas.User;
+
+public record UserData
+{
+    [JsonPropertyName("pid")]
+    public string Pid { get; init; }
+    [JsonPropertyName("name")]
+    public string? Name { get; init; }
+    [JsonPropertyName("surname")]
+    public string? Surname { get; init; }
+    [JsonPropertyName("birthdate")]
+    public string? Birthdate { get; init; }
+    [JsonPropertyName("birthNumber")]
+    public string? BirthNumber { get; init; }
+    [JsonPropertyName("mobileNumber")]
+    public string? MobileNumber { get; init; }
+    [JsonPropertyName("email")]
+    public string Email { get; init; }
```

Comment:
You could do other types of check with email and phone if they are valid etc.

Reply:
Ok I'll do this in this